ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/websocket/methods/PrivateWebSocketMethodsFactory.java

```diff
@@ -95,8 +94,8 @@ private PrivateMarkerTransactionFactory createPrivateMarkerTransactionFactory()
 
   private PrivacyController createPrivacyController() {
     final Optional<BigInteger> chainId = protocolSchedule.getChainId();
-    final DefaultPrivacyController defaultPrivacyController =
-        new DefaultPrivacyController(
+    final RestrictedDefaultPrivacyController restrictedDefaultPrivacyController =
```

Comment:
why for WS there is no Unrestricted controller?

Reply:
Good spot, thanks.